packages/core/src/lib/utils/logger.ts

```diff
@@ -34,18 +38,26 @@ const defaultMapConfig: LoggerMapConfig = {
   [LoggingScope.CORE]: {
     level: LoggingLevel.DEBUG,
     displayRequestId: false,
+    displayFilePath: 'hideNodeModulesOnly',
+    displayFunctionName: true,
   },
   [LoggingScope.BUILD]: {
     level: LoggingLevel.DEBUG,
     displayRequestId: false,
+    displayFilePath: 'hideNodeModulesOnly',
+    displayFunctionName: true,
   },
   [LoggingScope.SERVE]: {
     level: LoggingLevel.DEBUG,
     displayRequestId: false,
+    displayFilePath: 'hideNodeModulesOnly',
+    displayFunctionName: true,
```

Comment:
May I ask why we set different default config from Access log? 
```
2022-09-14 08:21:41.405  DEBUG [SERVE packages/serve/src/lib/response-formatter/jsonFormatter.ts:71 JsonStringTransformer.<anonymous>] convert to json format stream > done.
```
The path can be so long that I think it should be displayed with `--verbose` flag.

Reply:
Thanks for reviewing, discussing with me, and giving suggestions, I have made `CORE`, `BUILD`and , `SERVE` both hide the file path and function name, and according to the discussion, we will make it show `'hideNodeModulesOnly'` and display the function name in the future by implementing the `--verbose flag.

I also refactor the `getLogger`, so even non above the default logger scope also hides the file path and function name, it refactor by commit 9efb93e